src/components/InvertedFlatList/BaseInvertedFlatList.js

```diff
@@ -52,60 +46,66 @@ class BaseInvertedFlatList extends Component {
      *
      * @return {Object}
      */
-    getItemLayout(data, index) {
-        const size = this.sizeMap[index];
+    const getItemLayout = useCallback(
+        (data, index) => {
+            const size = sizeMap[index];
+
+            if (size) {
+                return {
+                    length: size.length,
+                    offset: size.offset,
+                    index,
+                };
+            }
+
+            // If we don't have a size yet means we haven't measured this
+            // item yet. However, we can still calculate the offset by looking
+            // at the last size we have recorded (if any)
+            const lastMeasuredItem = CollectionUtils.lastItem(sizeMap);
 
-        if (size) {
             return {
-                length: size.length,
-                offset: size.offset,
+                // We haven't measured this so we must return the minimum row height
+                length: initialRowHeight,
+
+                // Offset will either be based on the lastMeasuredItem or the index +
+                // initialRowHeight since we can only assume that all previous items
+                // have not yet been measured
+                offset: _.isUndefined(lastMeasuredItem) ? initialRowHeight * index : lastMeasuredItem.offset + initialRowHeight,
                 index,
             };
-        }
-
-        // If we don't have a size yet means we haven't measured this
-        // item yet. However, we can still calculate the offset by looking
-        // at the last size we have recorded (if any)
-        const lastMeasuredItem = CollectionUtils.lastItem(this.sizeMap);
-
-        return {
-            // We haven't measured this so we must return the minimum row height
-            length: this.props.initialRowHeight,
-
-            // Offset will either be based on the lastMeasuredItem or the index +
-            // initialRowHeight since we can only assume that all previous items
-            // have not yet been measured
-            offset: _.isUndefined(lastMeasuredItem) ? this.props.initialRowHeight * index : lastMeasuredItem.offset + this.props.initialRowHeight,
-            index,
-        };
-    }
+        },
+        [initialRowHeight, sizeMap],
```

Comment:
`sizeMap` shouldn't be in the dependencies because a ref won't produce a rerender, so if it changes a new function won't be created.

On the other hand, I don't see a need to memo this `getItemLayout`

Reply:
Honestly, I wasn't also convinced about using a memo here - but just left it to get your thoughts on it.